api/src/Entity/ActivityProgressLabel.php

```diff
@@ -43,6 +44,17 @@
         new GetCollection(
             security: 'is_authenticated()'
         ),
+        new GetCollection(
```

Comment:
For this operation, we should also add a test (similar to `testListCategoriesAsCampSubresourceIsAllowedForCollaborator`)